stable/mongodb-replicaset/README.md

```diff
@@ -55,6 +55,7 @@ The following table lists the configurable parameters of the mongodb chart and t
 | `securityContext.fsGroup`           | Group ID for the container                                                | `999`                                               |
 | `securityContext.runAsUser`         | User ID for the container                                                 | `999`                                               |
 | `securityContext.runAsNonRoot`      |                                                                           | `true`                                              |
+| `sidecarContainers`                 | Array of snippets with your sidecar containers for mongodb-replicasets    | `[]`                                                |
```

Comment:
I'd prefer `extraContainers`. The description should say something more like "Additional containers to add to the StatefulSet"